platform-integration/src/main/resources/com/softicar/platform/integration/database/structure/version/v30-structure.json

```diff
@@ -369,7 +369,7 @@
+    "createStatement": "CREATE TABLE `Workflow`.`WorkflowTransitionLog` (`workflowTransition` INT NOT NULL,`transaction` BIGINT NOT NULL,`name` VARCHAR(255) DEFAULT NULL,`htmlColor` VARCHAR(7) DEFAULT NULL,`active` BOOL DEFAULT NULL,`sourceNode` INT DEFAULT NULL,`targetNode` INT DEFAULT NULL,`notify` BOOL DEFAULT NULL,`autoTransition` BOOL DEFAULT NULL,`commentRequired` BOOL DEFAULT NULL,`requiredVotes` VARCHAR(255) DEFAULT NULL,`transitionIcon` INT DEFAULT NULL,`sideEffect` INT DEFAULT NULL,PRIMARY KEY (`workflowTransition`, `transaction`),KEY `transaction` (`transaction`),KEY `sourceNode` (`sourceNode`),KEY `targetNode` (`targetNode`),KEY `transitionIcon` (`transitionIcon`),KEY `sideEffect` (`sideEffect`),CONSTRAINT `WorkflowTransitionLog_ibfk_1` FOREIGN KEY (`workflowTransition`) REFERENCES `Workflow`.`WorkflowTransition` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,CONSTRAINT `WorkflowTransitionLog_ibfk_2` FOREIGN KEY (`transaction`) REFERENCES `Core`.`Transaction` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,CONSTRAINT `WorkflowTransitionLog_ibfk_3` FOREIGN KEY (`sourceNode`) REFERENCES `Workflow`.`WorkflowNode` (`id`) ON DELETE RESTRICT ON UPDATE RESTRICT,CONSTRAINT `WorkflowTransitionLog_ibfk_4` FOREIGN KEY (`targetNode`) REFERENCES `Workflow`.`WorkflowNode` (`id`) ON DELETE RESTRICT ON UPDATE RESTRICT,CONSTRAINT `WorkflowTransitionLog_ibfk_5` FOREIGN KEY (`transitionIcon`) REFERENCES `Workflow`.`WorkflowIcon` (`id`) ON DELETE RESTRICT ON UPDATE RESTRICT,CONSTRAINT `WorkflowTransitionLog_ibfk_6` FOREIGN KEY (`sideEffect`) REFERENCES `Core`.`Uuid` (`id`) ON DELETE RESTRICT ON UPDATE RESTRICT);"
```

Comment:
v30 must not be changed anymore because it was already released. This needs to be v31.